README.md

```diff
@@ -1,16 +1,16 @@
-# HPOlib2
+# HPOBench
 
-HPOlib2 is a library for hyperparameter optimization and black-box optimization benchmark with a focus on reproducibility.
+HPOBench is a library for hyperparameter optimization and black-box optimization benchmark with a focus on reproducibility.
 
-**Note:** Hpolib2 is under active construction. Stay tuned for more benchmarks. Information on how to contribute a new benchmark will follow shortly.
+**Note:** HPOBench is under active construction. Stay tuned for more benchmarks. Information on how to contribute a new benchmark will follow shortly.
 
-**Note:** If you are looking for a different version of HPOlib2, you might be looking for [HPOlib1.5](https://github.com/automl/HPOlib1.5) 
+**Note:** If you are looking for a different version of HPOBench, you might be looking for [HPOlib1.5](https://github.com/automl/HPOlib1.5) 
```

Comment:
Replace with "If you are looking for a different or older version of our benchmarking library, you might be looking for ..."